set.mm

```diff
@@ -519199,9 +519478,36 @@ such that every nonzero prime ideal contains a prime element (this is a
         XQXKXCYCOBURZAXKWTWRXCXMTXBXCUOXKXCYTOBCEWPGWOHJXDWDWEUNWHXFXPYAOBCEWPF
         XOHJXDWDWFUNAXPWIWTWRXCACDFEXOYQJIYSLWGTVJWJAXFWRUABURZAFGCUEXFUUASMUAB
         CEWPFGHJXDWKWLWMWN $.
+
+      $d .x. t $.  $d .|| t $.  $d B t $.  $d R t $.  $d U t $.  $d X t $.
+      $d Y t $.  $d ph t $.
+      rprmasso3.1 $e |- .x. = ( .r ` R ) $.
+      rprmasso3.u $e |- U = ( Unit ` R ) $.
+      $( In an integral domain, If a prime element divides another, they are
```

Comment:
"If" -> "if"

Reply:
Fixed in https://github.com/metamath/set.mm/pull/4855/commits/980eed034302056c78b3a07fdd6f0c21404cedbd